cmake/version_config.hpp.in

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define RAPIDSAI_RMM_VER_MAJOR @RMM_VERSION_MAJOR@
```

Comment:
Are we concerned with `RMM_VER_MAJOR` colliding with something else?

Reply:
Yes, safer than sorry. Also this repository name is rapidsai/rmm.

However, feel free to suggest a better name.

Reply:
A very minor comment, and maybe it conflicts with the convention in the rest of rapids, but since it's already such a long macro name, don't abbreviate `VERSION` -> `VER`, either. IMHO it's typical to find `...VERSION_{MAJOR,MINOR}` used in many projects.

Reply:
I renamed the macros to spell out `VERSION`.